profile/README.md

```diff
@@ -25,10 +25,11 @@ Explore every endpoint using the [Adyen Postman collections](https://www.postman
-|   [![Magento 2](https://raw.githubusercontent.com/devicons/devicon/1119b9f84c0290e0f0b38982099a2bd027a48bf1/icons/magento/magento-original.svg)](https://github.com/adyen-examples/adyen-magento-plugin-demo)   |  [Magento 2](https://github.com/adyen-examples/adyen-magento-plugin-demo)  | Payment plugin for frictionless online payments, in-app, and in-store, see also [Magento 2 sourcecode](https://github.com/Adyen/adyen-magento2).   |
-| [![Shopware 6](https://raw.githubusercontent.com/devicons/devicon/1119b9f84c0290e0f0b38982099a2bd027a48bf1/icons/shopware/shopware-original.svg)](https://github.com/adyen-examples/adyen-shopware-plugin-demo) | [Shopware 6](https://github.com/adyen-examples/adyen-shopware-plugin-demo) | Payment plugin for frictionless online payments, in-app, and in-store, see also [Shopware 6 sourcecode](https://github.com/Adyen/adyen-shopware6). |
+|                                                                                                        Plugin                                                                                                        |                                       Links                                       | Description                                                                                                                                        |
+|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|:---------------------------------------------------------------------------------:|:---------------------------------------------------------------------------------------------------------------------------------------------------|
+|     [![Magento 2](https://raw.githubusercontent.com/devicons/devicon/1119b9f84c0290e0f0b38982099a2bd027a48bf1/icons/magento/magento-original.svg)](https://github.com/adyen-examples/adyen-magento-plugin-demo)      |     [Magento 2](https://github.com/adyen-examples/adyen-magento-plugin-demo)      | Payment plugin for frictionless online payments, in-app, and in-store, see also [Magento 2 sourcecode](https://github.com/Adyen/adyen-magento2).   |
+| [![Headless Magento 2](https://raw.githubusercontent.com/devicons/devicon/1119b9f84c0290e0f0b38982099a2bd027a48bf1/icons/magento/magento-original.svg)](https://github.com/adyen-examples/magento-headless-demo) | [Headless Magento 2](https://github.com/adyen-examples/adyen-magento-plugin-demo) | A headless integration demo using Vue.js (frontend) and Magento 2 (backend).                                                                       |
```

Comment:
Please check the headless link link 

`[Headless Magento 2](https://github.com/adyen-examples/adyen-magento-plugin-demo) |`

Correct one: https://github.com/adyen-examples/magento-headless-demo

Reply:
Done